src/crypter.cpp

```diff
@@ -348,12 +355,10 @@ std::string AdvancedDecryptWithHWID(std::string data)
 
 std::string AdvancedCryptWithSalt(std::string boinchash, std::string salt)
 {
-
     try 
     {
        std::vector<unsigned char> vchSecret( boinchash.begin(), boinchash.end() );
-       std::string d1 = "                                                                                                                                        ";
-       std::vector<unsigned char> vchCryptedSecret(d1.begin(),d1.end());
+       std::vector<unsigned char> vchCryptedSecret;
```

Comment:
no space was allocated in that vector, is it OK?

Reply:
Yeah, the decrypter allocates the amount needed.

Reply:
Yeah, the encrypter allocates for source length + overhead, but I'll double check that it's actually correct.